packages/components/organisms/f-form/src/mixin/formValidation.mixin.js

```diff
@@ -0,0 +1,114 @@
+import { validationMixin } from 'vuelidate';
+import {
+    required, email
+} from 'vuelidate/lib/validators';
+import { validations } from '@justeat/f-services';
+import VueScrollTo from 'vue-scrollto';
+
+export default {
+    mixins: [validationMixin],
+
+    computed: {
+        fields () {
+            return this.formData.formFields;
+        },
+
+        validationState () {
+            return validations.getFormValidationState(this.$v);
+        },
+
+        invalidFieldsSummary () {
+            const invalidFieldCount = this.$v.formFields.$dirty
+                && validations.getFormValidationState(this.$v.formFields).invalidFields.length;
+
+            if (!invalidFieldCount) return null;
+
+            return this.$tc('invalidFields', invalidFieldCount);
+        }
+    },
+
+    validations () {
+        const invalidValidations = {
+            email,
+            mobileNumber: this.isValidPhoneNumber,
+            postcode: this.isValidPostcode
+        };
+
+        const formFields = {};
+
+        if (this.fields?.length) {
+            this.fields.forEach(field => {
+                const validationMessages = field.translations?.validationMessages;
+
+                if (validationMessages) {
+                    formFields[field.name] = {
+                        required,
+                        ...validationMessages.invalid && {
+                            isValid: invalidValidations[field.name]
+                        }
```

Comment:
I think generally we would style it like this instead:
```suggestion
                        ...(validationMessages.invalid ? {
                            isValid: invalidValidations[field.name]
                        } : {})
```